src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesToObject = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((acc, curr) => {
+      const [key, value] = curr.split(':').map((str) => str.trim());
```

Comment:
There is a potential issue here if `curr.split(':')` does not return exactly two elements. This can happen if the input string is malformed or if there are missing colons. Consider adding a check to ensure that both `key` and `value` are defined before attempting to trim and assign them.